src/Config/ObjectModel/RuntimeEntities.cs

```diff
@@ -67,7 +68,9 @@ public bool ContainsKey(string key)
             }
             else
             {
-                throw new ApplicationException($"The entity '{key}' was not found in the dab-config json");
```

Comment:
Till now, it was "keyNotFoundException" which got thrown when it tries to get an element from the dictionary. this code will just change the exception type and message. 
It came as feedback from an old PR. Since it is a small change, making it as part of this PR.